docker/hq-compose-citus.yml

```diff
@@ -7,23 +7,38 @@ services:
     labels: ['com.citusdata.role=Master']
     volumes:
       - ${VOLUME_PREFIX}citus_master:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: commcare_ucr_citus
```

Comment:
I think specifying the database is all that's needed to resolve https://github.com/dimagi/commcare-hq/pull/23479#discussion_r261684385

I was able to call citus up and can confirm that workers were connected to this database.

I'm guessing that the following was missed when configuring this intially:

>  It is based on the official PostgreSQL image, so be sure to consult that image’s documentation for advanced configuration options 

https://github.com/citusdata/docker/blob/master/README.md

Reply:
The DB name is not the issue at all. The issue is that if you do it this way then Django can't tear down the DB since it can't drop a DB that it's already connected to (regardless of what it's called). If you run this you'll see an error about Django being unable to connect to the `postgres` database during teardown.

The whole point of doing the setup in Django is so that for local testing the DB can be created and dropped by Django otherwise it requires manual intervention to remove the test database.

That's why in the original PR I had set `REUSE_DB=True` to prevent Django from trying to tear down the DB. If you look at the citusdata repo's they also use `reusedb` for this same reason: https://github.com/citusdata/django-multitenant/blob/master/pytest.ini#L3